jobs/sweep_schedulers/optuna_scheduler/optuna_scheduler.py

```diff
@@ -62,26 +62,26 @@ def setup_scheduler(scheduler: Scheduler, **kwargs):
     parser.add_argument("--entity", type=str, default=kwargs.get("entity"))
     parser.add_argument("--num_workers", type=int, default=1)
     parser.add_argument("--name", type=str, default=f"job-{scheduler.__name__}")
-    parser.add_argument("--enable_git", action="store_true", default=False)
     cli_args = parser.parse_args()
 
     settings = {"job_name": cli_args.name}
-    if cli_args.enable_git:
-        settings.update({"disable_git": True})
-
     run = wandb.init(
         settings=settings,
         project=cli_args.project,
         entity=cli_args.entity,
     )
     config = run.config
+    args = config.get("sweep_args", {})
 
-    if not config.get("sweep_args", {}).get("sweep_id"):
-        # not a sweep, just finish the run and return
+    if not args or not args.get("sweep_id"):
+        # when the config has no sweep args, this is being run directly from CLI
```

Comment:
So the user is running the `wandb launch -j <job>` here? Is this specifically done to create the job?

Reply:
now this would be triggered by running `python optuna_scheduler.py`